CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 1.2.0 / 2016-10-07
+
+* [FEATURE] Enable the use of query parameters in `/graph` URLs.
+* [FEATURE] PromQL: Add `minute()` function.
+* [FEATURE] Add GCE service discovery.
+* [FEATURE] Allow job names to start with a digit.
+* [FEATURE] Allow disabling local storage.
+* [FEATURE] EC2 service discovery: Expose `ec2_instance_state`.
+* [FEATURE] HTTP basic auth and TLS support for remote write path.
+* [FEATURE] Support for relabelling in remote write path.
+* [ENHANCEMENT] Add `go_import_path` to Travis so it works on a fork.
+* [ENHANCEMENT] Makefile: Promu installation now logged.
+* [ENHANCEMENT] Fingerprint locking: Avoid having contended mutexes on
+  same cacheline.
+* [ENHANCEMENT] Avoid slow `defer` in the `seriesMap.get` hot spot.
```

Comment:
Very implementation specific. Not every user will understand what this means for them in practice.
Possibly consolidate the performance improvements into a single understandable entry.